src/com/google/enterprise/adaptor/database/DatabaseAdaptor.java

```diff
@@ -378,14 +378,23 @@ private void addMetadata(MetadataHandler meta, ResultSet rs)
             }
           }
           break;
+        case Types.DATE:
+          Date dt = rs.getDate(index);
```

Comment:
s/Date/java.sql.Date/

This is currently a java.util.Date. No real badness, just a loss of clarity and temptation to misuse. Note that you can't import this, because we're importing java.util.Date.